Sources/macro.js

```diff
@@ -169,6 +169,8 @@ export function obj(publicAPI = {}, model = {}) {
     return subset;
   };
 
+  publicAPI.getOneProperty = val => model[val];
```

Comment:
Could it be replaced with the getReference macro? (I've added it while reviewing your code and reading your comment)

Reply:
I guess in that case it will just be a naming thing.

Reply:
Yes I think the implementation of getReference would work for this case.